src/rviz/ogre_helpers/movable_text.cpp

```diff
@@ -41,16 +41,16 @@
 
 #include "movable_text.h"
 
-#include <OGRE/OgreVector3.h>
-#include <OGRE/OgreQuaternion.h>
-#include <OGRE/OgreRoot.h>
-#include <OGRE/OgreCamera.h>
-#include <OGRE/OgreSceneNode.h>
-#include <OGRE/OgreMaterialManager.h>
-#include <OGRE/OgreHardwareBufferManager.h>
-#include <OGRE/Overlay/OgreFontManager.h>
-#include <OGRE/Overlay/OgreFont.h>
-#include <OGRE/OgreUTFString.h>
+#include <OgreVector3.h>
+#include <OgreQuaternion.h>
+#include <OgreRoot.h>
+#include <OgreCamera.h>
+#include <OgreSceneNode.h>
+#include <OgreMaterialManager.h>
+#include <OgreHardwareBufferManager.h>
+#include <Overlay/OgreFontManager.h>
```

Comment:
CMake will add Overlay to the include path, so you should not use the `Overlay/` prefix.

Also, there is native movable text in 1.12.10+:
https://www.ogre3d.org/2020/12/13/ogre-1-12-10-released#3dtext